src/pykx/pandas_api/pandas_meta.py

```diff
@@ -210,6 +234,27 @@ def abs(self, numeric_only=False):
             tab = _get_numeric_only_subtable(self)
         return q.abs(tab)
 
+    @api_return
+    def round(self, decimals: Union[int, Dict[str, int]] = 0):
```

Comment:
This function needs a couple of changes before it could be merged, at present it only handles `float` values but should probably also handle `real` values also (type -8h).

Reply:
After further reviewing, we could see that `round` is tricky, and we've decided to remove it from this contribution.